Bottom: fd793a5af83c9095d3fcabe6693e9f286a99d294
Top:    098cf985f3c169e8ac7e483f88ea60756ddf1a38
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2009-11-15 16:54:51 +0000

Refresh of move-gtk-drawing-routines-into

---

diff --git a/src/hid/gtk/gtkhid-gdk.c b/src/hid/gtk/gtkhid-gdk.c
index 77661ee..2f49da7 100644
--- a/src/hid/gtk/gtkhid-gdk.c
+++ b/src/hid/gtk/gtkhid-gdk.c
@@ -5,33 +5,11 @@
 #endif
 
 #include <stdio.h>
-#include <stdlib.h>
-#ifdef HAVE_STRING_H
-#include <string.h>
-#endif
-#include <math.h>
-#include <time.h>
-
 
-#include "global.h"
-#include "action.h"
 #include "crosshair.h"
-#include "data.h"
-#include "draw.h"
-#include "error.h"
-#include "mymem.h"
-#include "draw.h"
 #include "clip.h"
-
-#include "hid.h"
 #include "../hidint.h"
 #include "gui.h"
-#include "hid/common/draw_helpers.h"
-
-
-#if !GTK_CHECK_VERSION(2,8,0) && defined(HAVE_GDK_GDKX_H)
-#include <gdk/gdkx.h>
-#endif
 
 #ifdef HAVE_LIBDMALLOC
 #include <dmalloc.h>
diff --git a/src/hid/gtk/gtkhid-main.c b/src/hid/gtk/gtkhid-main.c
index aa41792..685330e 100644
--- a/src/hid/gtk/gtkhid-main.c
+++ b/src/hid/gtk/gtkhid-main.c
@@ -13,17 +13,9 @@
 #include <time.h>
 
 
-#include "global.h"
 #include "action.h"
 #include "crosshair.h"
-#include "data.h"
-#include "draw.h"
 #include "error.h"
-#include "mymem.h"
-#include "draw.h"
-#include "clip.h"
-
-#include "hid.h"
 #include "../hidint.h"
 #include "gui.h"
 #include "hid/common/draw_helpers.h"
